x-pack/qa/full-cluster-restart/src/test/resources/org/elasticsearch/xpack/restart/funny-timeout-watch.json

```diff
@@ -35,4 +35,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
guess this change can be discarded?

Reply:
reverting it back is hard inside the IDE, because the old invalidates editorconfig. I have to close the editor and edit it in vim to stop it automatically reverting back to the state in the PR. Leaving as is but good catch again!